components/GameCard.vue

```diff
@@ -0,0 +1,104 @@
+<script setup lang="ts">
+import OutlineHeartIcon from "@heroicons/vue/24/outline/HeartIcon"
```

Comment:
This is not gonna work, because there is no module "HeartIcon": it's already an icon to import. Here is the correct way how to do it:
```suggestion
import { HeartIcon as OutlineHeartIcon } from "@heroicons/vue/24/outline"
```